asv.conf.json

```
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "project": "tracksdata",
+  "project_url": "https://github.com/royerlab/tracksdata",
+  "repo": ".",
+  "benchmark_dir": "benchmarks",
+  "env_dir": ".asv/env",
+  "results_dir": ".asv/results",
+  "html_dir": ".asv/html",
+  "environment_type": "virtualenv",
```

```suggestion
  "environment_type": "uv",
```
Do you think this would work?